### PR TITLE
Format updates in systems topics folder (part 1)

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This is primarily as a fix for #108 (update formats in system folder).

- Converted remaining HTML tables to MD
- Replaced H2/H3 infinitives with present tense verb headings (better for machine translation and findability)
- Fixed levels for headings in some places
- Adjusted some procedures to better use numbered steps, as well as clarify navigation and simplify the instructions
- Added alt text for images where needed
- A few other misc. formats to align with MD linting rules

Note: To keep the PR from being too large, this is the first of a series to address #108

Also: This is not a comprehensive rewrite! Other style edits can be done at a later time.

## Affected documentation pages

https://docs.magento.com/m2/ee/user_guide/system/action-log-archive.html
https://docs.magento.com/m2/ee/user_guide/system/action-log-bulk-actions.html
https://docs.magento.com/m2/ee/user_guide/system/action-log-report.html
https://docs.magento.com/m2/ee/user_guide/system/action-log.html
https://docs.magento.com/m2/ee/user_guide/system/amazon-logs.html
https://docs.magento.com/m2/ee/user_guide/system/backups.html
https://docs.magento.com/m2/ee/user_guide/system/cache-full-page.html
https://docs.magento.com/m2/ee/user_guide/system/cache-management.html
https://docs.magento.com/m2/ee/user_guide/system/cron.html
https://docs.magento.com/m2/ee/user_guide/system/data-advanced-price.html
https://docs.magento.com/m2/ee/user_guide/system/data-attributes-customer.html
https://docs.magento.com/m2/ee/user_guide/system/data-attributes-product.html
https://docs.magento.com/m2/ee/user_guide/system/data-complex.html
https://docs.magento.com/m2/ee/user_guide/system/data-csv.html
https://docs.magento.com/m2/ee/user_guide/system/data-export-criteria.html
https://docs.magento.com/m2/ee/user_guide/system/data-export.html
https://docs.magento.com/m2/ee/user_guide/system/data-import-guidelines.html
https://docs.magento.com/m2/ee/user_guide/system/data-import-history.html
https://docs.magento.com/m2/ee/user_guide/system/data-import-price-tier.html
https://docs.magento.com/m2/ee/user_guide/system/data-import-product-images.html
https://docs.magento.com/m2/ee/user_guide/system/data-import.html
https://docs.magento.com/m2/ee/user_guide/system/data-schedule-export.html
https://docs.magento.com/m2/ee/user_guide/system/data-schedule-import.html
https://docs.magento.com/m2/ee/user_guide/system/data-scheduled-import-export.html
https://docs.magento.com/m2/ee/user_guide/system/data-transfer-configurable-products.html
https://docs.magento.com/m2/ee/user_guide/system/data-transfer-examples.html
https://docs.magento.com/m2/ee/user_guide/system/data-transfer-tax-rates.html
https://docs.magento.com/m2/ee/user_guide/system/data-validation.html
https://docs.magento.com/m2/ee/user_guide/system/system.html

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

## Additional information

Internal staging build 294

### CODE REVIEW

File: src/system/cache-management.md [hunk 1]
```diff
@@ -13,146 +13,135 @@ After clearing a cache, always refresh your browser to make sure that you can se
...
+- Always flush the cache after installing extensions/modules. You can install one or more extensions, then flush the cache.
+- Flush the cache after installing Magento Commerce. For fresh installs, you should also reindex.
+- Flush the cache after upgrading from one version of Open Source or Commerce to another.
+- When flushing caches, consider the type of cache and scheduling the flushing during non-peak times. For example, pick a time when few customers may access the site such as late night or early morning. Clearing some cache types during peak times cause result in a high load on the Admin and may result in a down site until completed.
+- When [reindexing]({% link system/index-management.md %}), you do not need to also perform a flush cache.
+  
+Access the Cache Management page by doing one of the following:
 
-#### To refresh specific caches:
+- Click the **Cache Management** link in the message above the workspace.
```
Comment (hickskenh) [inline src/system/cache-management.md]:
```suggestion
- Click **Cache Management** in the message above the workspace.
```

File: src/system/data-attributes-customer.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ The following tables list the attributes from a typical export of the Customers
 
 Each attribute, or field, is represented in the CSV file as a column, and customer records are represented by rows. Columns that begin with an underscore are service entities that contain properties or complex data.
 
-### Customers Main File
+## Customers Main File
```
Comment (hickskenh) [inline src/system/data-attributes-customer.md]: some of these fields are defined in devdocs.  the ones i found are below if you want to add them.

email - The customer's email address.
dob - The customer’s date of birth.
firstname - The customer’s first name.
gender - The customer’s gender.
group_id - The group assigned to the user. Default values are 0 (Not logged in), 1 (General), 2 (Wholesale), and 3 (Retailer).
lastname - The customer’s family name.
middlename - The customer’s middle name.
prefix - An honorific, such as Dr., Mr., or Mrs.
suffix - A value such as Sr., Jr., or III.
taxvat - The customer’s Tax/VAT number (for corporate customers).

Customer Addresses
city - The customer's city or town.
company - The customer's company.
country_id - The customer's country.
fax - The customer's fax number.
firstname - The first name of the person associated with the shipping/billing address.
lastname - The family name of the person associated with the shipping/billing address.
middlename - The middle name of the person associated with the shipping/billing address.
postcode - The customer’s ZIP or postal code.
prefix - An honorific, such as Dr., Mr., or Mrs.
region - An object that defines the customer’s state or province.
region_id - A number that uniquely identifies the state, province, or other area.
street - The number and name of the street associated with the shipping/billing address.
suffix - A value such as Sr., Jr., or III.
telephone - The customer's telephone number.
vat_id - The customer’s Tax/VAT number (for corporate customers).

File: src/system/data-attributes-product.md [hunk 3]
```diff
@@ -10,83 +10,87 @@ The installation used to export this data has the sample data installed, and has
 
 |Attribute|Description|
```
Comment (hickskenh) [inline src/system/data-attributes-product.md]: in the second table, the Attributes are in code.   do you want to make the first table the same?

File: src/system/data-complex.md [hunk 4]
```diff
@@ -4,13 +4,13 @@ title: Complex Data
...
+- [Grouped]({% link catalog/product-create-grouped.md %})
+- [Bundle]({% link catalog/product-create-bundle.md %})
 
 If you were to export a configurable product, you would find the standard attributes that make up a simple product, plus the additional attributes that are needed to manage complex data.
 
-![]({% link images/images/data-exported-configurable-product.png %}){: .zoom}
+![Configurable product - exported data]({% link images/images/data-exported-configurable-product.png %}){: .zoom}
 _Exported Configurable Product Data_
 
 ## Complex Data Attributes
```
Comment (hickskenh) [inline src/system/data-complex.md]: table Attributes in code?

File: src/system/data-export.md [hunk 5]
```diff
@@ -8,44 +8,44 @@ Special characters—such as the equal sign, greater and less than symbols, sing
...
+        ![Data export - exclude attributes]({% link images/images/data-export-filter-exclude.png %}){: .zoom}
         _Exclude Attributes_
 
-1.  Scroll down and click **Continue** in the lower-right corner of the page.
+1. Scroll down and click **Continue** in the lower-right corner of the page.
 
-1.  Upon completion of the task, look for the download prompt in the lower-right corner of your browser.
+1. Upon completion of the task, look for the file in the download location for your web browser.
 
     You can save or open the exported CSV file as a spreadsheet, then edit the data and import it back into your store.
```
Comment (hickskenh) [inline src/system/data-export.md]: suggestion:   add Import topic link to "import" in the last sentence.

[import]({% link system/data-import.md %})

File: src/system/data-import-guidelines.md [hunk 6]
```diff
@@ -4,26 +4,26 @@ title: Import Guidelines
 
 ## New Entities
 
--   Entities are added with the attribute values specified in the CSV file.
--   If there is no value, or there is a non-valid value, for a required attribute with no default value set, then the entity (the corresponding row or rows) cannot be imported.
--   If there is no value, or there is a non-valid value, for a required attribute with the default value set, then the entity (the corresponding row or rows) is imported, and the default value is set for the attribute.
--   If the complex data is not valid, then the entity (the corresponding row or rows) cannot be imported.
+- Entities are added with the attribute values specified in the CSV file.
+- If there is no value, or there is a non-valid value, for a required attribute with no default value set, then the entity (the corresponding row or rows) cannot be imported.
```
Comment (hickskenh) [inline src/system/data-import-guidelines.md]:
```suggestion
- If there is no value or there is a non-valid value, for a required attribute with no default value set, then the entity (the corresponding row or rows) cannot be imported.
```

File: src/system/data-import-guidelines.md [hunk 7]
```diff
@@ -4,26 +4,26 @@ title: Import Guidelines
...
+- If there is no value, or there is a non-valid value, for a required attribute with the default value set, then the entity (the corresponding row or rows) is imported, and the default value is set for the attribute.
```
Comment (hickskenh) [inline src/system/data-import-guidelines.md]:
```suggestion
- If there is no value or there is a non-valid value, for a required attribute with the default value set, then the entity (the corresponding row or rows) is imported, and the default value is set for the attribute.
```

File: src/system/data-import-guidelines.md [hunk 8]
```diff
@@ -4,26 +4,26 @@ title: Import Guidelines
...
+- If the complex data is not valid, then the entity (the corresponding row or rows) cannot be imported.
 
 ## Existing Entities
 
--   For attributes that are not complex data, the values from the import file, including the empty values for the non-required attributes, replace the existing values.
--   If there is no value, or there is a non-valid value, for a required attribute, then the existing value is not replaced.
--   If the complex data for the entity is invalid, the entity (the corresponding row or rows) cannot be imported, except the case, when Delete Entities was selected in the Import Behavior drop-down menu.
+- For attributes that are not complex data, the values from the import file, including the empty values for the non-required attributes, replace the existing values.
+- If there is no value, or there is a non-valid value, for a required attribute, then the existing value is not replaced.
```
Comment (hickskenh) [inline src/system/data-import-guidelines.md]:
```suggestion
- If there is no value or there is a non-valid value, for a required attribute, then the existing value is not replaced.
```

File: src/system/data-import-price-tier.md [hunk 9]
```diff
@@ -4,90 +4,84 @@ title: Importing Tier Prices
...
-1.  Update the tier price data for each product, as needed.
+1. Update the tier price data for each product, as needed.
 
-1.  Any products without tier price updates can be deleted from the CSV file. There is no need to reimport products that haven’t changed.
+    Any products without tier price updates can be deleted from the CSV file. There is no need to reimport products that have not changed.
 
-1.  **Save** the updated CSV file.
+1. **Save** the updated CSV file.
 
     {:.bs-callout-info}
```
Comment (hickskenh) [inline src/system/data-import-price-tier.md]: i disagree with this change.  when looking at a file size, it is a number, not text.

File: src/system/data-import-price-tier.md [hunk 10]
```diff
@@ -4,90 +4,84 @@ title: Importing Tier Prices
...
+1. On _Admin_ sidebar, go to **System** > _Data Transfer_ > **Import**.
 
-1.  Under _Data Transfer_, choose **Import**.
+1. Under _Import Settings_, set **Entity Type** to `Advanced Pricing`.
 
-1.  Under _Import Settings_, set **Entity Type** to `Advanced Pricing`.
+1. Set **Import Behavior** to `Add/Update`.
 
-1.  Set **Import Behavior** to `Add/Update`.
+1. Under **File to Import**, click **Choose File** and select the file that you prepared to import from your directory.
```
Comment (hickskenh) [inline src/system/data-import-price-tier.md]:
```suggestion
1. Under _File to Import_, click **Choose File** and select the file that you prepared to import from your directory.
```

File: src/system/data-import-product-images.md [hunk 11]
```diff
@@ -25,17 +24,19 @@ We recommend that you use a program that supports UTF-8 encoding to edit CSV fil
     {: .bs-callout-info }
     Starting with the Magento `2.3.2` release, the path specified in the **Images File Directory** concatenates for import to the images base directory - `<Magento root folder>/var/import/images`. For earlier Magento releases, you can use a different folder on the Magento server, as long as the path to the folder is specified during the import process.
```
Comment (hickskenh) [inline src/system/data-import-product-images.md]: suggest removing "release", just leaving Magento 2.3.2.

File: src/system/data-export.md [hunk 12]
```diff
@@ -8,44 +8,44 @@ Special characters—such as the equal sign, greater and less than symbols, sing
```
Comment (hickskenh) [inline src/system/data-export.md]: suggestion:   below is the special character paragraph copied from the Import topic.  use same formatting/wording for consistency?

Special characters (such as the equal sign, greater and less than symbols, single and double quotes, backslash, pipe, and ampersand symbols) can cause problems during data transfer. To ensure that such special characters are correctly interpreted, they can be marked as an _escape sequence_. For example, if the data includes a string of text such as `code=”str”`, `code=”str2”`, choosing to enclose the text in double quotes ensures that the original double quotes are understood to be part of the data. When the system encounters a double set of double quotes, it understands that the outer set of double quotes is enclosing the actual data.

File: src/system/data-schedule-export.md [hunk 13]
```diff
@@ -19,76 +19,76 @@ After each export, the export file is placed in the user-defined location, and a
...
-    ![]({% link images/images-ee/data-transfer-scheduled-export-file-information.png %}){: .zoom}
+    ![Scheduled export file information]({% link images/images-ee/data-transfer-scheduled-export-file-information.png %}){: .zoom}
     _Export File Information_
 
-1.  Enter the **File Directory** where the export file is to be saved as follow:.
+1. Enter the **File Directory** where the export file is to be saved as follow:.
 
-    -  For Local Server, enter a relative path within the Magento installation. For example, `var/export`.
-    -  For Remote FTP server, enter the full URL and path to the target folder on the destination server.
+    - For Local Server, enter a relative path within the Magento installation. For example, `var/export`.
```
Comment (hickskenh) [inline src/system/data-schedule-export.md]:
```suggestion
    - For **Local Server**, enter a relative path within the Magento installation. For example, `var/export`.
```

File: src/system/data-schedule-export.md [hunk 14]
```diff
@@ -19,76 +19,76 @@ After each export, the export file is placed in the user-defined location, and a
...
+    - For Remote FTP server, enter the full URL and path to the target folder on the destination server.
```
Comment (hickskenh) [inline src/system/data-schedule-export.md]:
```suggestion
    - For **Remote FTP server**, enter the full URL and path to the target folder on the destination server.
```

File: src/system/data-scheduled-import-export.md [hunk 15]
```diff
@@ -5,22 +5,20 @@ title: Scheduled Import/Export
 
 Scheduled imports and exports can be run on a daily, weekly or monthly basis. The files to be imported or exported can be located on local Magento servers, or on remote FTP servers. Scheduled Import/Export is implemented by default, and does not require additional configuration. All scheduled imports and exports are managed by the Cron job scheduler.
```
Comment (hickskenh) [inline src/system/data-scheduled-import-export.md]: daily, weekly, or monthly....

File: src/system/data-transfer-configurable-products.md [hunk 16]
```diff
@@ -6,83 +6,81 @@ The best way to understand how configurable product data is structured, is to ex
...
-        * Multi-Select
+        - `Dropdown`
+        - `Visual Swatch`
+        - `Text Swatch`
+        - `Multi-Select`
 
-    - Values Required—yes
+    - **Values Required** — `Yes`
 
 1. If you are adding a new size or color, or making any other change to an existing attribute, make sure to update the attribute with the new value.
```
Comment (hickskenh) [inline src/system/data-transfer-configurable-products.md]: color or making...

File: src/system/data-transfer-configurable-products.md [hunk 17]
```diff
@@ -6,83 +6,81 @@ The best way to understand how configurable product data is structured, is to ex
...
     - Set **Entity Type** to `Products`.
 
     - Set **Export File Format** to `CSV`.
 
-    ![]({% link images/images/data-transfer-export-settings.png %}){: .zoom}
+    ![Data export settings]({% link images/images/data-transfer-export-settings.png %}){: .zoom}
     _Export Settings_
 
-1. Under **Entity Attributes**, scroll down to **SKU**. Then, do the following:
+1. Under **Entity Attributes**, scroll down to **SKU** and do the following:
```
Comment (hickskenh) [inline src/system/data-transfer-configurable-products.md]:
```suggestion
1. Under _Entity Attributes_, scroll down to **SKU** and do the following:
```

File: src/system/data-transfer-configurable-products.md [hunk 18]
```diff
@@ -100,78 +98,88 @@ _New Size Variation Added for Each Color_
...
+1. If the file is valid, click **Import**.
+
+   Otherwise, correct any problems found in the data and try again.
 
-    ![]({% link images/images/data-transfer-configurable-product-import-validation-results.png %}){: .zoom}
+    ![System message - file is valid]({% link images/images/data-transfer-configurable-product-import-validation-results.png %}){: .zoom}
     _Validation Results_
 
-1. When the import is complete, click the **Cache Management** link in the message at the top of the page. Then, refresh all invalid caches.
+1. When the import is complete, click the **Cache Management** link in the message at the top of the page and refresh all invalid caches.
```
Comment (hickskenh) [inline src/system/data-transfer-configurable-products.md]:
```suggestion
1. When the import is complete, click **Cache Management** in the message at the top of the page and refresh all invalid caches.
```

File: src/system/data-transfer-tax-rates.md [hunk 19]
```diff
@@ -6,74 +6,82 @@ If you conduct business in several states and ship a large quantity of product,
...
+    - Rate
+    - Range From
+    - Range To
+    - A column for each store view
+
+    ![Exported data - tax rates]({% link images/images/data-exported-tax-rates.png %}){: .zoom} 
     _Exported Tax Rate Data_
 
-1.  Open the new tax rate data in a second instance of the spreadsheet, so you can see both, side by side.
+1. Open the new tax rate data in a second instance of the spreadsheet, so you can see them side by side.
```
Comment (hickskenh) [inline src/system/data-transfer-tax-rates.md]: did you mean to restart numbering here?

File: src/system/data-transfer-tax-rates.md [hunk 20]
```diff
@@ -6,74 +6,82 @@ If you conduct business in several states and ship a large quantity of product,
...
-1.  Under _Data Transfer_, choose **Import/Export Tax Rates**.
+1. Click **Export Tax Rates**.
 
-1.  Click **Export Tax Rates**. Look for the download file in the lower-left corner of the browser window.
+1. Look for the file in the download location for your web browser.
 
-1.  Save the file, and then open it in a spreadsheet. (In this example, we use OpenOffice Calc.)
+1. Save the file, and then open it in a spreadsheet.
+
+  This example uses OpenOffice Calc.
```
Comment (hickskenh) [inline src/system/data-transfer-tax-rates.md]: this entire list is displaying as code, along with the image link, no image.
https://docs.magedevteam.com/294/m2/ee/user_guide/system/data-transfer-tax-rates.html

File: src/system/data-validation.md [hunk 21]
```diff
@@ -4,49 +4,49 @@ title: Data Validation
...
--  **Complex Data**—Values that originate from a defined set, such as a drop-down or multiple select input type, are verified to ensure that the values exist in the defined set.
--  **Service Data**—The values in service data columns are verified to ensure that the properties or complex data values are consistent with what is already defined in the system database.
--  **Required Values**—For new entities, the presence of required attribute values in the file is checked. For existing entities, there is no need to re-check the existence of required attribute values.
--  **Separators**—Although the separators aren’t visible when viewed in a spreadsheet, data values in a CSV file are separated by comma, and text values are enclosed in double-quotes. During the validation process, the separators are verified, and each set of quotes that enclose character strings is verified to be formatted correctly.
+- **Attributes** - Column header names are verified to ensure that they match the corresponding attributes in the system database. The value of each attribute is checked to ensure that it meets the requirements of the data type (decimal, integer, varchar, text, and datetime)
```
Comment (hickskenh) [inline src/system/data-validation.md]:
```suggestion
- **Attributes** - Column header names are verified to ensure that they match the corresponding attributes in the system database. The value of each attribute is checked to ensure that it meets the requirements of the data type (decimal, integer, varchar, text, and datetime).
```

File: src/system/system.md [hunk 22]
```diff
@@ -6,7 +6,7 @@ sections:
    url: /system/data-transfer.html
```
Comment (hickskenh) [inline src/system/system.md]: the tabs seem to have different text formats.  some start with Learn how to....others start with the task (Set Up Admin rights).

File: src/system/action-log-report.md [hunk 23]
```diff
@@ -11,61 +11,60 @@ The report can be filtered using the options in each column. You may set a singl
 
 The Action Logs Report includes the following information:
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/system/action-log-report.md]: That is a good question.... I did not do any editing for these. I think we need to add to styles issue list. If it is not a complete sentence, does it need a period?

File: src/system/amazon-logs.md [hunk 24]
```diff
@@ -6,20 +6,22 @@ The Amazon Pay extension collects and provides viewable and downloadable logs fo
...
+## Display and Download Amazon Logs
 
-1.  On the _Admin_ sidebar, click **Stores**.
+1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-1.  Under _Settings_, choose **Configuration**.
+1. In the left panel, expand **Sales** and choose **Payment Methods**.
 
-1.  Expand ![]({% link images/images/btn-expand.png %}) the **Advanced** section, then expand **Developer Options**.
+1. For **Amazon Pay**, click <span class="btn">Configure</span>.
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/system/amazon-logs.md]: Good suggestion

File: src/system/system.md (hunk 22 above)
Comment (leslietilling) [reply src/system/system.md]: I think that is kind of a good thing that they are varied a little bit that way... although I would say I don't really like the "learn how to" phrasing for things unless it is a tutorial.

File: src/system/cache-full-page.md [hunk 25]
```diff
@@ -9,49 +9,47 @@ We recommend Varnish to be used only in a production environment.
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/system/cache-full-page.md]: done

File: src/system/backups.md [hunk 26]
```diff
@@ -2,43 +2,45 @@
...
 
--   **Restrict access to backup tools.** 
-    Access to the Backups and roll back management tool can be restricted by configuring user permissions for backup and roll back resources. To restrict access, leave the corresponding checkbox unselected. If you need to grant access to roll back resources, you must grant access to backup resources as well.
+- **Restrict access to backup tools.** Access to the Backups and roll back management tool can be restricted by configuring user permissions for backup and roll back resources. To restrict access, leave the corresponding checkbox unselected. If you need to grant access to roll back resources, you must grant access to backup resources as well.
```
[the remark this answers is not in the document]
Comment (leslietilling) [reply src/system/backups.md]: done

File: src/system/data-transfer-tax-rates.md (hunk 19 above)
Comment (leslietilling) [reply src/system/data-transfer-tax-rates.md]: fixed

File: src/system/data-validation.md (hunk 21 above)
Comment (leslietilling) [reply src/system/data-validation.md]: done

File: src/system/data-transfer-tax-rates.md (hunk 20 above)
Comment (leslietilling) [reply src/system/data-transfer-tax-rates.md]: fixed

**Comment by leslietilling** [comment]
running tests

File: src/system/action-log-report.md (hunk 23 above)
[the remark this answers is not in the document]
Comment (hickskenh) [reply src/system/action-log-report.md]: i was going off of formats in other topics.  seems options lists use periods.

**Pull request merged**